include/dt-bindings/sensor/ina23x.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2021 The Chromium OS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_INA23X_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_INA23X_H_
+
+/*
+ * Provide the BIT() macro for when this file is included from
+ * devicetrees.
+ */
+#ifndef BIT
+#define BIT(n) (1 << (n))
+#endif
```

Comment:
would likely be better to make it available to DT

Reply:
Could you elaborate on this? I'm not sure what you mean.

Reply:
sounds like a feature that would be useful to have on DT, so instead of providing a local solution, adjust dt utils.

Reply:
Got it. Thanks

Reply:
I think this is just a matter of moving BIT from util.h to util_macro.h. The latter is available in DT.

Reply:
Yes I agree. I'll move the BIT macros from util.h to util_macro.h. Should this be done in a separate pull request?

Reply:
I think it's OK to put it as the first commit in this PR as prep work personally